### PR TITLE
Update config validation

### PR DESCRIPTION
**Pull request by thiagokroger**

### CODE REVIEW

**Comment by thiagokroger** [comment]
@zimny I created a new PR with the branch on the main repo, jut one question left, you mentioned to check the config file before deploying.

Would that be here
https://github.com/nearform/mira/blob/2132c449714e863ea0bdb2555f7c26d2d7687c68/src/cdk/bootstrap.ts#L318-L319
and here
https://github.com/nearform/mira/blob/2132c449714e863ea0bdb2555f7c26d2d7687c68/src/cdk/bootstrap.ts#L335

Or inside of each function?

**Comment by zimny** [comment]
@thiagokroger Yes, before deploy functions

File: src/cdk/bootstrap.ts [hunk 1]
```diff
@@ -339,6 +346,17 @@ export class MiraBootstrap {
         }
         break
       case 'cicd':
+        if (!this.args.file) {
```
Comment (zimny) [inline src/cdk/bootstrap.ts:349]: cicd does not accept a file parameter afaik, from where this requirement comes from?

Comment (thiagokroger) [reply src/cdk/bootstrap.ts]: Good catch, I did by mistake, just fixed.

**Pull request merged**